ffmpeg-sys-the-third/build.rs

```diff
@@ -769,17 +769,15 @@ fn check_features(include_paths: &[PathBuf]) {
         }
     }
 
-    let version_check_info = [("avcodec", 57, 62, 0, 101)];
-    for &(lib, begin_version_major, end_version_major, begin_version_minor, end_version_minor) in
-        &version_check_info
-    {
-        let libversion = *versions
-            .get(lib)
-            .expect("Unable to find the version for lib{lib}");
-
+    let begin_version_major = 57;
+    let end_version_major = 62;
+    let begin_version_minor = 0;
+    let end_version_minor = 134;
+    for (lib, version) in &versions {
+        println!("lib{lib} version = {}_{}", (*version).0, (*version).1);
```

Comment:
Out of curiosity: What do you need the exact library version for?

Reply:
This was just for debugging purposes